docs/src/learning_rate_scheduler.md

```diff
@@ -13,23 +13,25 @@ We shall compare the following configurations of implemented schedulers.
 4. Misfit controlling timestep (Terminating) `DataMisfitController()` [Iglesias & Yang 2021](https://doi.org/10.1088/1361-6420/abd29b) - purple
 5. Misfit controlling timestep (Continuing beyond Terminate condition) `DataMisfitController(on_terminate="continue")` - brown
 
-One can define the schedulers as
-```julia
-scheduler = DefaultScheduler(0.5) # fixed stepsize, default values: 1
-```
-Then when constructing an EnsembleKalmanProcess, one uses the keyword argument
+!!! info "Recommended Scheduler"
```

Comment:
I agree that this is the best scheduler to use for getting the posterior but may not be the best to start with, e.g., someone's first calibration run, because the total number of iterations is not defined. I'm thinking particularly about expensive forward models.

Reply:
One suggestion could be to use this scheduler and then set a `max_iter` (assuming there's a kwarg for this?) Then the user could indicate roughly how much resources they're willing to allocate for an initial calibration.

Reply:
So for this scheduler the user still submits a max iteration, and here it really will be an upper bound on the iterations. The only difference is that this will prevent further updates if the number of required iterations go beyond the end time. The benefit is the calibrations are much more robust, and have more interpretation than the fixed timestep one, and due to the adaptivity are typically more efficient.

However I'll think about whether there is a nicer message/hints that we can do to aid first-time users.

Reply:
I've created a message to indicate that we return `true` from  `update_ensemble!(` I also flushed out the LRS docs.